qiskit/quantum_info/operators/symplectic/clifford.py

```diff
@@ -147,56 +173,102 @@ def __str__(self):
+
+    @paulis.setter
+    def paulis(self, value):
+        """Set the Pauli list"""
+        if not isinstance(value, PauliList):
+            value = PauliList(value)
+        self._paulis._z[:, :] = value._paulis._z
+        self._paulis._x[:, :] = value._paulis._x
+        self._paulis._phase[:] = value._paulis._phase
+
+    # TODO: DEPRECATE table, stabilizer, and destabilizer
+    @property
+    # @deprecate_function(
+    #    "The Clifford.table method is deprecated as of Qiskit Terra 0.19.0 "
+    #    "and will be removed no sooner than 3 months after the releasedate. "
+    #    "Use Clifford.paulis method instead.",
+    # )
     def table(self):
         """Return StabilizerTable"""
-        return self._table
+        return StabilizerTable(
+            np.column_stack((self.paulis.x, self.paulis.z)), phase=self.paulis.phase // 2
+        )
 
     @table.setter
+    # @deprecate_function(
+    #    "The Clifford.table method is deprecated as of Qiskit Terra 0.19.0 "
+    #    "and will be removed no sooner than 3 months after the releasedate. "
+    #    "Use Clifford.paulis method instead.",
+    # )
     def table(self, value):
         """Set the stabilizer table"""
         # Note this setter cannot change the size of the Clifford
         # It can only replace the contents of the StabilizerTable with
         # another StabilizerTable of the same size.
         if not isinstance(value, StabilizerTable):
             value = StabilizerTable(value)
-        self._table._array[:, :] = value._table._array
-        self._table._phase[:] = value._table._phase
+        self.paulis(value)
 
     @property
+    # @deprecate_function(
+    #    "The Clifford.stabilizer method is deprecated as of Qiskit Terra 0.19.0 "
+    #    "and will be removed no sooner than 3 months after the releasedate. "
+    #    "Use Clifford.paulis method instead.",
+    # )
     def stabilizer(self):
         """Return the stabilizer block of the StabilizerTable."""
-        return StabilizerTable(self._table[self.num_qubits : 2 * self.num_qubits])
+        return StabilizerTable(self.table[self.num_qubits : 2 * self.num_qubits])
 
     @stabilizer.setter
+    # @deprecate_function(
+    #    "The Clifford.stabilizer method is deprecated as of Qiskit Terra 0.19.0 "
+    #    "and will be removed no sooner than 3 months after the releasedate. "
+    #    "Use Clifford.paulis method instead.",
+    # )
     def stabilizer(self, value):
         """Set the value of stabilizer block of the StabilizerTable"""
         inds = slice(self.num_qubits, 2 * self.num_qubits)
-        self._table.__setitem__(inds, value)
+        self._paulis.__setitem__(inds, value)
 
     @property
+    # @deprecate_function(
+    #    "The Clifford.destabilizer method is deprecated as of Qiskit Terra 0.19.0 "
+    #    "and will be removed no sooner than 3 months after the releasedate. "
+    #    "Use Clifford.paulis method instead.",
+    # )
     def destabilizer(self):
```

Comment:
This should be updated to return PauliList of destabilizer rows instead of StabilizerTable

Reply:
I think that a `destabilizer` method is still useful since otherwise we should call:
`clifford.paulis[0:num_qubits]` which is more complicated.
(especially in the` StabilizerState` and decomposition)